packages/terra-popup/src/terra-dev-site/test/popup/ArrowHorizontalAttachmentsPopup.test.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/jsx-curly-brace-presence */
```

Comment:
why are we disabling all these?

Reply:
Resolved problem by adding a aria-label here: [d678858](https://github.com/cerner/terra-framework/pull/1166/commits/d67885871dee3d05aa83267625adaa0b9134ccc5).